ncm-ceph/src/main/perl/Ceph/ClusterMap.pm

```diff
@@ -0,0 +1,220 @@
+#${PMpre} NCM::Component::Ceph::ClusterMap${PMpost}
+
+use 5.10.1;
+
+use parent qw(CAF::Object);
+use Readonly;
+use Data::Dumper;
+use JSON::XS;
+
+
+sub _initialize
+{
+    my ($self, $clusterobj) = @_;
+
+    $self->{log} = $clusterobj;
+
+    $self->{quattor} = {};
+    $self->{ceph} = {};
+
+    $self->{actions} = {
```

Comment:
will this ever change? if not, make it a readonly. also, why is this a map at all? the method name is a straight map with the keys?